projects/angular/src/data/tree-view/_tree-view.clarity.scss

```
@@ -161,6 +162,17 @@
         color: tree-view-variables.$clr-tree-node-content-active-color;
       }
     }
+
+    // Add touch area to label that covers the checkbox, so we don't need to resize the
+    // checkbox itself, which leads to undesired side effects.
+    &::before {
+      content: '';
+      position: absolute;
+      left: 0;
+      top: 0;
+      transform: translateX(-100%);
```

Could we do something like that instead?
```suggestion
      left: calc(-1 * tree-view-variables.$clr-tree-node-touch-target);
      top: 0;
```